tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -634,7 +635,10 @@ def fit(self,
                 to any amount of labeled data.
             verbosity : int
                 If specified, overrides the existing `predictor.verbosity` value.
-
+            calibrate: bool, default = False
+                If True then will use temperature scaling to calibrate the model for better negative log loss. Temperature scaling
+                will train a scalar parameter on the validation set. This trains the best model's predictive prob to have a stronger
```

Comment:
Delete this sentence or make it more clear: "This trains the best model's predictive prob to have a stronger correlation between predictive prob and accuracy."